### ISSUE TITLE
DayView ignores AllDay attribute

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bradrydzewski/gwt-cal#33]
```
DayView does not take the Appointment.AllDay property into account when 
placing appointments in the multi-day header.

Need to update the DayView to use the same Multi/AllDay logic that MonthView 
uses.

I'd like to fix this prior to the 0.9 final release 
```

Original issue reported on code.google.com by `Brad.Ryd...@gmail.com` on 4 Mar 2010 at 3:41

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r287.
```

Original comment by `Brad.Ryd...@gmail.com` on 29 Mar 2010 at 3:02
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This was fixed in the gwt-cal 0.9 branch
```

Original comment by `Brad.Ryd...@gmail.com` on 29 Mar 2010 at 3:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `Brad.Ryd...@gmail.com` on 12 May 2010 at 7:06
- Added labels: **Release0.9**
